dawn/renderer/components/Editor.js

```diff
@@ -29,10 +27,27 @@ import 'brace/theme/solarized_dark';
 import 'brace/theme/solarized_light';
 import 'brace/theme/terminal';
 
+import UpdateBox from './UpdateBox';
+import ConfigBox from './ConfigBox';
+import ConsoleOutput from './ConsoleOutput';
+import EditorButton from './EditorButton';
+
+const Client = require('ssh2').Client;
+
 const dialog = remote.dialog;
 const currentWindow = remote.getCurrentWindow();
 
 class Editor extends React.Component {
+  static pathToName(filepath) {
```

Comment:
You can move generic functions like this into a `utility.js` file and import wherever you need.